odl/discr/discr_mappings.py

```diff
@@ -138,15 +138,15 @@ def order(self):
         return self._order
 
 
-class RawGridCollocation(FunctionSetMapping):
+class GridCollocation(FunctionSetMapping):
 
     """Function evaluation at grid points.
 
-    This is the raw `Operator` version of the default 'restriction'
-    used by all core discretization classes.
+    This is the default 'restriction' used by all core
```

Comment:
We need a glossary or similar where users can look up words like 'restriction', perhaps not in this pull though.